Packs/EWSMailSender/Integrations/EWSMailSender/EWSMailSender.py

```diff
@@ -123,6 +123,16 @@ def get_account(account_email):
 
 def send_email_to_mailbox(account, to, subject, body, bcc=None, cc=None, reply_to=None,
                           html_body=None, attachments=[], raw_message=None, from_address=None):
+    """
+    Send an email to a mailbox.
+
+    Args:
+        account (Account): account from which to send an email.
+        to (list[str]): a list of emails to send an email.
```

Comment:
isnt this python3? lets put that as `, to: List[str]` instead.
Or if you mean its a comma delimited list, write that instead. Seems misleading otherwise

Reply:
you can use both ways, i am not a huge fan of type annotations, i like to stick with stating the types in the docstrings. 
On this case ill do both of them.